docs/readmes/contributing/contribute_workflow.md

```diff
@@ -27,7 +27,7 @@ Once your PR has been approved and passed all CI checks, use the [`ready2merge`]
 
 - [Supported commit types](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json):
 `build`, `chore`, `ci`, `docs`, `feat`, `fix`, `perf`, `refactor`, `revert`, `style`, `test`
-- [Supported scopes](https://github.com/magma/magma/blob/master/.github/semantic.yml): `agw`, `orc8r`, `nms`, `feg`, and more
+- [Supported scopes](https://github.com/magma/magma/blob/a7ff21e6144ece504cbf2e78d51d319b54b8a391/.github/workflows/semantic-pr.yml): `agw`, `orc8r`, `nms`, `feg`, and more
```

Comment:
Good catch! Can we keep this to point to master instead of the commit hash?

Reply:
@themarwhal updated!